src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -45,13 +49,18 @@
   /** Store class that gives access to Users. */
   private UserStore userStore;
 
+  /** Store class that gives access to Mentions. */
+  private UserStore MentionStore;
```

Comment:
for the word "MentionStore" make the 'M' lowercase also would the type be "MentionStore" instead of UserStore?